src/utils.py

```diff
@@ -2439,3 +2449,21 @@ def nostdout() -> Generator[None, None, None]:
     sys.stdout = _DummyFile()
     yield
     sys.stdout = save_stdout
+
+
+def query_ldl(ldl: LiftedDecisionList, atoms: Set[GroundAtom],
+              goal: Set[GroundAtom]) -> Optional[_GroundNSRT]:
```

Comment:
why do we return the ground NSRT instead of the ground rule?

Reply:
I'm thinking about this like querying a (partial) policy in the abstract space. also, I don't foresee the rule being useful except to get the ground NSRT from it